aya-obj/src/generated/btf_internal_bindings.rs

```diff
@@ -1,4 +1,4 @@
-/* automatically generated by rust-bindgen 0.65.1 */
+/* automatically generated by rust-bindgen 0.68.1 */
```

Comment:
Please take the generated/* changes out of the PR, we generate the bindings in CI

Reply:
Done, sorry!